src/python/turicreate/toolkits/_tf_utils.py

```diff
@@ -58,6 +58,26 @@ def convert_conv1d_coreml_to_tf(conv_weights):
 	conv_weights = np.transpose(conv_weights, (3, 1, 0, 2))
 	return np.squeeze(conv_weights, axis=3)
 
+def convert_conv2d_coreml_to_tf(conv_weights):
+	
+	"""
+	The Convolutional weights in CoreML specification converted to 
+	the TensorFlow format for training in TensorFlow.
+
+	Parameters
+    ----------
+    conv_weights: 4d numpy array of shape 
+       [outputChannels, kernelChannels, kernelHeight, kernelWidth] 
+
+    Returns
+    -------
+	return: 4d numpy array of shape
+	   [kernelWidth, kernelHeight, kernelChannels, outputChannels]
+
+	"""
+	conv_weights = np.transpose(conv_weights, (3, 2, 0, 1))
```

Comment:
If the input is OIHW, doesn't a `(3, 2, 0, 1)` transpose yield WHOI, not WHIO?

And isn't the TF format HWIO? https://www.tensorflow.org/api_docs/python/tf/nn/conv2d